blocks/extensions/__init__.py

```diff
@@ -128,6 +128,24 @@ def on_interrupt(self):
         pass
 
 
+class CallbackName(str):
```

Comment:
There it's `which_callback`.

Anyway, if there are `callback` occurrences, we should consider switching to `callback_name`.